identity-service/src/relay/txRelay.js

```diff
@@ -107,7 +107,7 @@ const sendTransactionInternal = async (req, web3, txProps, reqBodySHA) => {
     where: { walletAddress: req.body.senderAddress },
     attributes: ['blockchainUserId']
   })
-  const userId = user.blockchainUserId
+  const userId = user == null ? "unknown" : user.blockchainUserId
```

Comment:
i don't think this would catch undefined which it is in this case. maybe change to 
```
const userId = user?.blockchainUserId ?? "unknown";
```